extensions/packageManager/src/client/Library.tsx

```diff
@@ -7,6 +7,7 @@ import React, { useState, Fragment, useEffect } from 'react';
 import formatMessage from 'format-message';
```

Comment:
The render part of this component has become very big, consider breaking it down into smaller components in the future.

Reply:
https://github.com/microsoft/BotFramework-Composer/issues/5830